internal/resources/diffValidation/resource_monitor.go

```diff
@@ -0,0 +1,68 @@
+//  (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+
```

Comment:
can u name this file as resource_nsxt_lb_monitor ?

Reply:
done